### ISSUE TITLE
Proposal to change the "clear" icon in the plot and viewer-and

### ISSUE THREAD

**Issue by thomasp85** [posit-dev/positron#4458]
The current icon used to clear the plot and viewer pane is not that telling since it alludes to a context consisting of text. I would propose to change it to something like below (quick mock-up) to make its intend more clear.

The icon could be further improved by adding "<>" to the center for the viewer pane, and a linechart to the plot pane if we wanted even more clarity

<img width="527" alt="image" src="https://github.com/user-attachments/assets/e255ce5e-4ae3-46f8-944d-f9444123e864">

**Comment by paul-dicristina** [comment]
We're going to use the clear symbol from RStudio adapted to the Positron style: 
<img width="69" alt="image" src="https://github.com/user-attachments/assets/ad1aee1c-5753-4b50-b328-d30cb5488103">
It's preferable to have one symbol that works across contexts.

**Comment by jthomasmock** [comment]
Howdy @paul-dicristina ?, we just came back around on this one in triage.

Can we do the same shape/square as currently but just remove the lines so it looks less like text?

**Comment by paul-dicristina** [comment]
That icon is not one of mine - it’s part of the standard codeicon set.
I think whoever implemented the chart pane stuck that in there and ignored the clear plot icon that I supplied.

![Image](https://github.com/user-attachments/assets/d612046e-4367-433b-a6cf-e4e0f91c2d10)

**Comment by juliasilge** [comment]
Thank you, @paul-dicristina! I added that in https://github.com/posit-dev/positron-codicons/pull/11 and tried that out in the plots pane:

<img width="590" height="608" alt="Image" src="https://github.com/user-attachments/assets/5f7dcc8d-210e-4d25-a285-1018e64f5710" />

It looks quite thick compared to the other icons being used in that action bar. Do you think we could get a new "thin" version of this icon?

**Comment by paul-dicristina** [comment]
![Image](https://github.com/user-attachments/assets/8ad355df-c0ec-43fe-bd36-5887944039fc)

**Comment by juliasilge** [comment]
Are you happy with this, @paul-dicristina? Especially how this icon looks next to the "Open plots gallery in new window" icon?

<img width="606" height="180" alt="Image" src="https://github.com/user-attachments/assets/2c489c0e-9e21-4723-9000-d1df1526b7ad" />


-------

For reference, here is where we're starting from:

<img width="511" height="111" alt="Image" src="https://github.com/user-attachments/assets/9c5a2f90-a255-4bd3-b71d-6c5dac811d57" />

If it's helpful here is:

- what we're using for "gallery in new window" there: https://github.com/posit-dev/positron-codicons/blob/main/src/icons/window.svg
- the current "clear" icon we're using: https://github.com/posit-dev/positron-codicons/blob/main/src/icons/clear-all.svg

**Comment by paul-dicristina** [comment]
<img width="405" height="230" alt="Image" src="https://github.com/user-attachments/assets/ca9e64a4-4551-4823-9ed9-c7af864ac1e1" />

**Comment by paul-dicristina** [comment]
I would recommend these.

![Image](https://github.com/user-attachments/assets/56ee9fb8-ae7f-44d9-a6aa-6b8a23f31e25)
![Image](https://github.com/user-attachments/assets/78945cd9-1488-4871-a7dc-765a54a65789)

**Comment by paul-dicristina** [comment]
I think all of the clear icons with the small X attribute are not great because the X is simply too hard to parse.
It works in RStudio because we have color to distinguish it but Positron is lacking in that dimension.

**Comment by juliasilge** [comment]
We addressed and made these consistent in https://github.com/posit-dev/positron/pull/13776.
